pkg/apis/serving/v1beta1/component.go

```diff
@@ -80,6 +80,14 @@ type ComponentExtensionSpec struct {
 	// Maximum number of replicas for autoscaling.
 	// +optional
 	MaxReplicas int `json:"maxReplicas,omitempty"`
+	// ScaleTarget specifies scaling value of the component
+	// concurrency(https://knative.dev/docs/serving/autoscaling/autoscaling-targets/).
+	// +optional
+	ScaleTarget *int `json:"scaleTarget,omitempty"`
+	// ScaleMetric specifies scaling metric of the component
+	// concurrency(https://knative.dev/docs/serving/autoscaling/autoscaling-metrics/).
+	// +optional
+	ScaleMetric *string `json:"scaleMetric,omitempty"`
```

Comment:
@andyi2it Can we define an enum for the scaling metric type with kubebuilder annotation like https://github.com/kserve/kserve/blob/5091f0f87dfc0ecd91aff4722b603686ca53349b/pkg/apis/serving/v1beta1/inference_service_status.go#L154, so the value can be validated by validating webhook with openapi schema